service/src/main/kotlin/app/cash/backfila/dashboard/CloneBackfillAction.kt

```diff
@@ -63,19 +64,7 @@ class CloneBackfillAction @Inject constructor(
     private val connectorProvider: ConnectorProvider
 ) : WebAction {
 
-  @Post("/backfills/{id}/clone")
-  @RequestContentType(MediaTypes.APPLICATION_JSON)
-  @ResponseContentType(MediaTypes.APPLICATION_JSON)
-  // TODO allow any user
-  @Authenticated(capabilities = ["users"])
-  fun create(
-      @PathParam id: Long,
-      @RequestBody request: CloneBackfillRequest
-  ): CloneBackfillResponse {
-    // TODO check user has permissions for this service with access api
-
-    logger.info { "Clone backfill $id by ${caller.get()?.user}" }
-
+  fun validate(request: CloneBackfillRequest) {
```

Comment:
moved stuff around in CloneAction, still want to reuse stuff from Create later

Reply:
Yeah that's my main comment. Why can't this use the backfill creator object?

Reply:
it can, but I wanted to keep the pr smaller. I should have just not changed this file

Reply:
I guess you are going to do a follow up? I'm fine with that.

Reply:
yep. the exact shared api is still unclear to me